x-pack/plugins/infra/public/alerting/metric_anomaly/components/expression.test.tsx

```diff
@@ -59,6 +59,7 @@ describe('Expression', () => {
         setRuleProperty={() => {}}
         metadata={currentOptions}
         dataViews={dataViewMock}
+        onChangeMetaData={() => {}}
```

Comment:
This event doesn't seem to be used in the `Expression` component. [Should we omit ](https://github.com/elastic/kibana/blob/main/x-pack/plugins/infra/public/alerting/metric_anomaly/components/expression.tsx#L40)`onChangeMetaData` instead, to avoid confusing the expectations for this component?